src/app/services/BD/index.ts

```diff
@@ -39,3 +40,37 @@ export const getSelectedOffice = async (id: string) => {
       return null;
     });
 };
+
+export const setReservation = async (placeReservation: PlaceReservation) => {
+  const result = new Promise<string>((resolve, reject) => {
+    firebase
+      .database()
+      .ref(
+        "offices/" +
+          placeReservation.idOffice +
+          "/floors/" +
+          placeReservation.selectFloor +
+          "/places/" +
+          placeReservation.indexPlace +
+          "/placeStatus/"
+      )
+      .set(
```

Comment:
is it possible to use `.get()` after setting? maybe this will allow to avoid using our Promise. Check something like this:
```
return await firebase
  .database()
  .ref(
        "offices/" +
          placeReservation.idOffice +
          "/floors/" +
          placeReservation.selectFloor +
          "/places/" +
          placeReservation.indexPlace +
          "/placeStatus/"
      )
      .set(...)
      .get()
```  
just to know if it works